docker-compose.yml

```diff
@@ -1,114 +1,99 @@
 # first build or re-build:
-# docker-compose up --build
+# docker-compose build --build-arg GIT_HASH $(git rev-parse --short HEAD) --parallel
+# full re-build with no cache from previous buils:
+# docker-compose build --build-arg GIT_HASH $(git rev-parse --short HEAD) --parallel --no-cache
 # otherwise:
-# docker-compose up
+# docker-compose up --detach
+# to stop:
+# docker-compose down
 
 version: "3.1"
 
 services:
-
   server:
     container_name: polis-server
     build:
       context: ./server
       dockerfile: Dockerfile
-    environment:
-      STATIC_FILES_HOST: file-server
-      STATIC_FILES_PORT: 8080
-      STATIC_FILES_ADMINDASH_PORT: 8080
-      SHOULD_USE_TRANSLATION_API: 'false'
-      DATABASE_URL: postgres://postgres:oiPorg3Nrz0yqDLE@postgres:5432/polis-dev
-      WEBSERVER_USERNAME: ws-user
-      WEBSERVER_PASS: ws-pass
-      MATH_ENV: dev
+    env_file: ./server/docker-dev.env
```

Comment:
What about passing a single file into all services? And what are your thoughts on `.env` with explicit passing of specific vars into each service?

Reply:
That could be fine. Occasionally there can be a conflict, like if multiple services have an env var for `PORT` or `HOST`. I try not to pass env into a service that isn't using them just in case.

Reply:
Ok! Anyhow I'm +1 on single file, but no block in this PR :)